src/webln/NostrWeblnProvider.ts

```diff
@@ -283,6 +294,19 @@ export class NostrWebLNProvider implements WebLNProvider, Nip07Provider {
       (result) => ({ paymentRequest: result.invoice }),
     );
   }
+
+  lookupInvoice(args: LookupInvoiceArgs) {
+    this.checkConnected();
+
+    return this.executeNip47Request<LookupInvoiceResponse>(
+      "lookup_invoice",
+      "lookupinvoice",
```

Comment:
do we actually need to pass the two here? or can this be some mapping? 
this is only needed for the events, isn't it?

Reply:
@bumi the webln request method is needed for the notification function. WebLN and NWC event names are inconsistent, I think we need to provide both.

Unless we can guarantee the only change is in casing, then we could have a mapping function from camelCase to lower_case_with_underscores

Reply:
but can't we do a mapping from 

```
a = { 
  "lookup_invoice" : "lookupinvoice" 
}
a["lookup_invoice"]
```

that then can be checked?